app/Schedule/View.elm

```diff
@@ -198,40 +164,76 @@ laterTrainView now train =
         ]
 
 
-prediction : Date -> Train -> Node Msg
-prediction now train =
+prediction : Date -> Direction -> Train -> Node Msg
+prediction now direction train =
     let
-        predictedDeparture =
+        actualDeparture =
             Maybe.withDefault train.scheduledDeparture train.predictedDeparture
 
+        minutesUntilDeparture =
+            minutesBetween now actualDeparture
+
         minutesLate =
-            predictedMinutesLateText <|
-                minutesFrom train.scheduledDeparture predictedDeparture
+            minutesBetween train.scheduledDeparture actualDeparture
+
+        minutesLateString =
+            predictedMinutesLateText minutesLate
 
         predictionString =
-            predictionText now minutesLate predictedDeparture
+            joinMaybeWith ", "
```

Comment:
Doing this means we get away with removing the comma from the "late" string below, too.